_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,210 @@
+
+2. After the transaction was confirmed, as part of a group of transactions contained in a newly-mined block being relayed across the network.
+
+There's no need for the node to receive the transaction a second time if it still has the first copy. Compact block relay (BIP152) can eliminate this redundancy by allowing a node to receive from its peers an ordered list of what transactions are included in a new block. With this knowledge, the node can use the transactions it has already received to partly or fully reconstruct the transactions part of the block for itself.  If the node doesn't receive all the transactions it needs to fully reconstruct the block, it requests the missing transactions from its peers, and then uses them to complete the block.
+
+![Compact Blocks diagram](https://raw.githubusercontent.com/bitcoin/bips/master/bip-0152/protocol-flow.png)
+
+Compact blocks provides three very important benefits to the network:
+
+1. By reducing the amount of bandwidth used by transaction relay nodes, compact blocks help offset the expected increase in bandwidth that will occur when the segwit capacity increases are enabled by miners.  This offset should allow nodes to continue operating on the network after segwit even if they're presently near their current bandwidth caps.
+
+2. By eliminating the bandwidth spike that occurs when nodes receive a new block, compact blocks may make it easier to keep a node operating on connections with limited peak bandwidth.  For example, several users have reported that receiving a new block slows down other important activity on their network, such as video conferencing, so some of those users shut down Bitcoin Core before starting those activities.  Compact blocks may eliminate those bandwidth spikes and make running Bitcoin Core less inconvenient for those users.
+
+3. Faster propagation of blocks across the network, by dramatically reducing the amount of data that needs to be transported when a new block is found.
+
+**More information:**
+
+- [Release notes][release notes compact blocks]
+
+- [Compact Block Relay FAQ](/en/2016/06/07/compact-blocks-faq/)
+
+- [BIP152][], which describes the compact blocks protocol
+
+- [Bitcoin FIBRE](http://bitcoinfibre.org/), an open source protocol and implementation that builds upon compact blocks to minimize latency for new block announcements between peers on managed networks.  FIBRE was designed and implemented alongside compact block relay version 1 and it is being used to test improvements for a subsequent version of compact block relay.
+
+## Fee filtering
+
+For several years now, Bitcoin Core nodes have used a minimum relay fee rate to determine what unconfirmed transactions they'll process, relay, and store in their individual memory pools.  Each node gets to decide its own minimum relay fee rate, and if they receive a transaction whose fee rate is below that limit, they don't add it to their memory pools or relay it to their other peers.
```

Comment:
@harding Yeah I can appreciate that the nuance here is hard to get right for a general audience!  On further thought, I think your text is fine as-is -- the details here are probably not so important to anyone.

Since I started this comment though I might as well clarify the technology to make sure we're all on the same page: When an 0.13.0 node is started up, no fee filter messages are sent, no matter what you set your minrelaytxfee to.  It's not until the mempool fills up and some stuff gets evicted from it that a node will start sending feefilter messages.

So no bandwidth is saved until the mempool fills up.  Once the mempool does fill up, then it is true that the feefilter message will be generated, and its value will be at least as large as your minrelaytxfee (or higher, if transactions with feerates higher than your minrelayfee are getting evicted from the mempool).  But I'd imagine that if you set your minrelaytxfee to be very high, then you'd not save any bandwidth at all, because your mempool would never get full, so you'd never send a feefilter message.

Given that (a) mempools are generally full, and (b) we're even contemplating changing the behavior to be more along the lines of what you described anyway (maybe even for 0.13.1), the micro-detail about this only kicking in currently when mempools are full is not so important.

Thanks for writing this up!